connector/connect/docs/client-connection-string.md

```diff
@@ -81,6 +82,15 @@ sc://hostname:port/;param1=value;param2=value
     <pre>user_id=Martin</pre>
     </td>
   </tr>
+  <tr>
+    <td>user_agent</td>
+    <td>String</td>
+    <td>The user agent acting on behalf of the user, typically applications
+    that use Spark Connect to implement its functionality and execute Spark
+    requests on behalf of the user.<br/>
+    <i>Default: </i>_SPARK_CONNECT_PYTHON</td>
```

Comment:
It's only the default in the Python client. :) The proto itself is probably just empty.

Reply:
+1